addons/website/views/snippets/s_rating.xml

```diff
@@ -2,17 +2,17 @@
 <odoo>
 
 <template id="s_rating" name="Rating">
-    <div class="s_rating pt16 pb16" data-vcss="001" data-icon="fa-star">
-        <h4 class="s_rating_title">Quality</h4>
+    <div class="s_rating s_rating_no_title pt16 pb16" data-vcss="001" data-icon="fa-star" aria-label="3 out of 5 stars">
+        <h4 class="s_rating_title">Rating</h4>
         <div class="s_rating_icons o_not_editable">
-            <span class="s_rating_active_icons">
-                <i class="fa fa-star"/>
-                <i class="fa fa-star"/>
-                <i class="fa fa-star"/>
+            <span class="s_rating_active_icons" style="color: var(--warning)">
```

Comment:
Is warning a good idea? :thinking: Maybe hardcoding the color would be better no? Or at best use `var(--yellow)` (for once bootstrap "plain colors" might make sense?)